README.rst

```diff
@@ -13,3 +13,131 @@ __ https://discuss.openedx.org/c/working-groups/build-test-release/30
+
+================================
+Role definitions and assignments
+================================
+
+*Current release: Lilac*
+
+The following roles, with corresponding expectations and assignments, are
+defined for the current release cycle.
+
+Group Chair
+===========
+
+*Current assignee: @arbrandes*
+
+The group chair is responsible for coordinating efforts that culminate in
+getting a new release out and maintaining the current one.  In particular, the
+Group Chair will:
+
+* Manage the current release cycle in general, including making sure that
+  proper deadlines are established and adhered to
+* Facilitate the assignment of roles and tasks
+* Coordinate the group's communication efforts, including running periodic
+  meetings such as the Biweekly Meetup, and summarizing recent happenings in
+  the Biweekly Update
+* Try and help solve problems as they come up, bringing them to the attention
+  of the right group members
+
+Release Manager
+===============
+
+*Current assignees: @nedbat and @arbrandes*
+
+The Release Manager is responsible for the actual cutting of a release, which,
+in essence, includes following the steps in the `Process to Create an Open edX Release
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/19662426/Process+to+Create+an+Open+edX+Release>`_.
+
+In addition, the Release Manager will:
+
+* Create public test or release candidate branches as required by the release process
+* Merge reviewed PRs into the release master branch (for instance,
+  ``koa.master``)
+* Notify the Release Testing Coordinator whenever additional testing is needed
+  in response to the above
+* Tag releases, major and minor, at the appropriate time
+
+Release Testing Coordinator
+===========================
+
+*Current assignee: @jfavellar90*
+
+The Release Testing Coordinator is responsible for making sure release
+candidates are tested _before_ a tag is made.  The coordinator does not
+necessarily have to run tests themselves, but they will:
+
+* Coordinate testing in general, in particular in the days leading to a timed
+  release, including the bimonthly minor ones.
+* Keep tabs on whatever CI is available, making sure it is running correctly
+* Advocate for better testing tools and environments, when necessary
+* Coordinate the reaction to test failures, such as bringing them to the
+  attention of the rest of the group by opening tickets or commenting in the forum
+
+Release Documentation Expert
+============================
+
+*Current assignee: N/A*
+
+The Release Documentation Expert is tasked with building the Release Notes
+document prior to a major release.  It involves:
+
+* Tracking down all relevant information,
+* Writing a draft document and posting it for community review with
+sufficient lead time prior to a release
+* Adjusting it as per the former review, and finally releasing a final version
+  concomittantly with the major release itself
```

Comment:
@nedbat can you add some links or specify the tools used for drafting and publishing the release notes (unless that's too specific for this documentation)?

Reply:
The release notes are in the edx-documentation repo, published on ReadTheDocs.  The drafts are often composed and edited in Google Docs.  Is that what you mean?

Reply:
Yep. That's what I thought the process was, but I wanted to confirm, for my own purposes. If it's appropriate to add that detail here, I think that would be a good idea.

Reply:
I guess the "pre-PR" collaboration tool is up to whoever is leading the writing, so it's up to you, @pdpinch.  Personally, I'd skip the doc and just do everything in a WIP PR. 🤷🏼 

In ay case, I added a link to the [edx-documentation](https://github.com/edx/edx-documentation/tree/master/en_us/open_edx_release_notes/source) repo, as you suggested.